models/Document/Hardlink.php

```diff
@@ -18,7 +18,7 @@
 
 use Pimcore\Model;
 use Pimcore\Model\Document;
-use Pimcore\Model\Redirect;
+use Pimcore\Bundle\SeoBundle\Model\Redirect;
```

Comment:
The doDelete will break if the bundle is not installed. 
Maybe we can also add a listener for this?

Reply:
This will still break. :)